source/extensions/filters/http/local_ratelimit/local_ratelimit.h

```diff
@@ -53,6 +66,9 @@ class FilterConfig : public Router::RouteSpecificFilterConfig {
   ~FilterConfig() override = default;
   const LocalInfo::LocalInfo& localInfo() const { return local_info_; }
   Runtime::Loader& runtime() { return runtime_; }
+  const envoy::extensions::filters::http::local_ratelimit::v3::LocalRateLimit protoConfig() const {
+    return proto_config_;
```

Comment:
`const envoy::extensions::filters::http::local_ratelimit::v3::LocalRateLimit&`? Do we need a full copy of the proto?

Reply:
I had this as a reference in the beginning but I believe you had pointed out that the reference could go stale:

> I think we need to copy the proto object, since this reference could (will) go stale.

Either way, I'm now storing the individual data we need under `FilterConfig` and have gotten rid of the full copy of the proto but now we're in a situation where we're individually storing more or less the same amount of data in `FilterConfig` maybe with a few exceptions ...